internal/storage/postgres_test.go

```diff
@@ -97,4 +99,55 @@ func TestPGMeta(t *testing.T) {
 			}
 		})
 	}
+
+	// testing out upsertStmt
+	ctx := context.Background()
+
+	upsertTests := []struct {
+		tableName   string
+		expectedSQL string
+	}{
+		{
+			tableName:   "table1",
+			expectedSQL: `INSERT INTO table1(0,jason,big ben) VALUES ($1,$2,$3) ON CONFLICT (id) DO UPDATE SET "0" = EXCLUDED."0","jason" = EXCLUDED."jason","big ben" = EXCLUDED."big ben"`,
+		},
+		{
+			tableName:   "table2",
+			expectedSQL: `INSERT INTO table2(1,john,bakers street) VALUES ($1,$2,$3) ON CONFLICT (id,name) DO UPDATE SET "1" = EXCLUDED."1","john" = EXCLUDED."john","bakers street" = EXCLUDED."bakers street"`,
+		},
+		{
+			tableName:   "table3",
+			expectedSQL: `INSERT INTO table3(2,harry,leicester square) VALUES ($1,$2,$3) ON CONFLICT (id,name,address) DO UPDATE SET "2" = EXCLUDED."2","harry" = EXCLUDED."harry","leicester square" = EXCLUDED."leicester square"`,
+		},
+	}
+
+	expectedSQL := ""
+	actualSQL := ""
```

Comment:
```suggestion
```

If we move the mock function, we wont need these anymore. See my comment about that.